django_freeradius/base/models.py

```diff
@@ -399,11 +399,26 @@ class AbstractRadiusAccounting(models.Model):
     framed_ip_address = models.GenericIPAddressField(verbose_name=_('framed IP address'),
                                                      db_column='framedipaddress',
                                                      db_index=True,
-                                                     # the default MySQL freeradius schema defines
-                                                     # this as NOT NULL but defaulting to empty string
-                                                     # but that wouldn't work on PostgreSQL
```

Comment:
Did you mean to remove this comment?

Reply:
yes because it no longer as not null in freeradius schema

Reply:
https://github.com/FreeRADIUS/freeradius-server/blob/master/raddb/mods-config/sql/main/postgresql/schema.sql#L41

Reply:
@Weirdo914, it's still NOT NULL in the mysql schema: https://github.com/FreeRADIUS/freeradius-server/blob/master/raddb/mods-config/sql/main/mysql/schema.sql#L42, although it defaults to empty string but the information in the comment is still useful to remember why it's done this way. Please put back this comment.

Reply:
Done